appium-cookies/lib/cookies.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
+ */
+import _ from 'underscore';
+
+export default function cookies(cookieString) {
```

Comment:
I think we should rewrite this as an ES6 class, as follows:

```js
export default class Cookies {
  constructor (cookieString) {
  }
  ...
}
```

It will require re-thinking a few sections but it's the kind of idiom we are using now.